test/types/findByPk.ts

```diff
@@ -19,4 +22,69 @@ import { expectTypeOf } from 'expect-type';
   expectTypeOf(await User.findByPk(Buffer.from('asdf'), {
     rejectOnEmpty: new Error('')
   })).toEqualTypeOf<User>();
+
+  // raw
+
+  expectTypeOf(await User.findByPk(Buffer.from('asdf'), {
+    raw: true,
+  })).toEqualTypeOf<Attributes<User> | null>();
+
+  expectTypeOf(await User.findByPk(Buffer.from('asdf'), {
+    raw: false,
+  })).toEqualTypeOf<User | null>();
+
+  expectTypeOf(await User.findByPk(Buffer.from('asdf'), {
+    raw: undefined,
+  })).toEqualTypeOf<User | null>();
+
+  // combination
```

Comment:
Do we want to combine the undefined as well?

Reply:
Same for findOne

Reply:
Done